src/instructlab/model/evaluate.py

```diff
@@ -1,12 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=C0412
```

Comment:
Can you include the rule name here instead of the code?

Reply:
sure